src/Microsoft.DotNet.Interactive.Kql/KqlConnection.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Kusto.Data;
+using Kusto.Data.Security;
+using Microsoft.DotNet.Interactive.Connection;
+using Microsoft.DotNet.Interactive.SqlServer;
+
+namespace Microsoft.DotNet.Interactive.Kql
+{
+    public class KqlConnection : KernelConnection
```

Comment:
The naming would be a bit clearer I think if this were `KqlKernelConnection`.

Reply:
done

Reply:
done